src/components/Gizmo.tsx

```diff
@@ -42,15 +42,15 @@ export default function Gizmo() {
     scene.add(...gizmoAxes, ...gizmoAxisHeads)
 
     const clock = new Clock()
-    const clientCamera = SceneInfra.instance.camControls.camera
+    const clientCamera = sceneInfra.camControls.camera
     let currentQuaternion = new Quaternion().copy(clientCamera.quaternion)
 
     const animate = () => {
       requestAnimationFrame(animate)
       updateCameraOrientation(
         camera,
         currentQuaternion,
-        clientCamera.quaternion,
+        sceneInfra.camControls.camera.quaternion,
```

Comment:
this was a subtle bug where currentCamera would go stale because we swap the camera out for a orthographic camera when going into a sketch, so it would have a reference to a dead camera, currentCamera was stuck in the closure, but using the full `sceneInfra.camControls.camera.quaternion` in the animate function fixed this.

This bug only show up when entering and exiting a sketch and then the gizmo would stop updating.

Reply:
My apologies for missing 

> Another thing is that the gizmo stops working after we exit the sketch plane.
Shall I check on it while we are waiting?

Sorry this was the problem.